core-web/libs/portlets/edit-ema/portlet/src/lib/edit-ema-editor/components/dot-uve-toolbar/components/dot-ema-bookmarks/dot-ema-bookmarks.component.html

```diff
@@ -1,12 +1,16 @@
 @if (store.$previewMode()) {
     <p-button
+        pTooltip="Bookmark"
+        tooltipPosition="bottom"
```

Comment:
Take a look at the Doc and check if there exists a way to set this globally with `constructor(private primengConfig: PrimeNGConfig) {}` [config](https://v17.primeng.org/configuration).

Reply:
I haven't found a way to make it globally yet!